pkg/actuators/machine/utils.go

```diff
@@ -96,6 +97,15 @@ func conditionFailed() metav1.Condition {
 	}
 }
 
+func conditionBuild() metav1.Condition {
```

Comment:
some way can we enhance to test case to see this condition is getting added or not?